includes/Modules/Thank_With_Google/Web_Tag.php

```diff
@@ -103,33 +122,35 @@ protected function render() {
 	 * @since 1.80.0
 	 */
 	protected function enqueue_twg_script() {
-		$twg_src = 'https://news.google.com/thank/js/v1/thank.js';
+		$is_singular = $this->is_singular_cta_post_type_entity();
+		$is_static   = $this->has_static_cta_placement();
+
+		$subscription = array(
+			'type'              => 'Blog',
+			'isPartOfType'      => array( 'Blog', 'Product' ),
+			'isPartOfProductId' => $this->tag_id . ':default',
+			'postTitle'         => $is_singular ? get_the_title() : '',
+			'permalink'         => $is_singular ? get_permalink() : '',
+			'pluginVersion'     => GOOGLESITEKIT_VERSION,
+			'colorTheme'        => $this->color_theme,
+			'promptSettings'    => array(
+				'style' => $is_static ? 'inline' : 'floating',
+			),
+		);
 
-		$is_singular_cta_post_type_entity = $this->is_singular_cta_post_type_entity();
+		if ( self::PLACEMENT_DYNAMIC_LOW === $this->cta_placement ) {
+			$subscription['promptSettings']['floatingPromptProminence'] = 'low';
+		} elseif ( self::PLACEMENT_DYNAMIC_HIGH === $this->cta_placement ) {
+			$subscription['promptSettings']['floatingPromptProminence'] = 'high';
+		}
 
 		$twg_inline_script = sprintf(
-			"
-			(self.SWG_BASIC = self.SWG_BASIC || []).push(subscriptions => {
-				subscriptions.init({
-					type: 'Blog',
-					isPartOfType: ['Blog', 'Product'],
-					isPartOfProductId: '%s:default',
-					buttonPosition: '%s',
-					permalink: '%s',
-					pluginVersion: '%s',
-					postTitle: '%s'
-				});
-			});
-			",
-			esc_js( $this->tag_id ),
-			esc_js( $this->has_static_cta_placement() ? 'inline' : 'floating' ),
-			esc_js( $is_singular_cta_post_type_entity ? get_permalink() : '' ),
-			esc_js( GOOGLESITEKIT_VERSION ),
-			esc_js( $is_singular_cta_post_type_entity ? get_the_title() : '' )
+			'(self.SWG_BASIC=self.SWG_BASIC||[]).push(function(s){s.init(%s);})',
```

Comment:
This is a minor point, but I am not so sure about the change from `subscriptions =>` to `function(s)` here. It seems we've lost a bit of readability. I would just keep it as it was. Although, if serving less bytes is the point of it, we could use `s=>` instead and then add a comment to explain what `s` is. What do you think?

Reply:
@techanvil, the standard `function(...) {...}` callbacks have a slightly better support by older browsers than arrow functions. I thought that it wouldn't hurt if we used `function(s) {...}` instead of `(s) => {...}` in order to make sure that this script wouldn't fail on older browsers.

Reply:
I can revert it to be an arrow function again. Just let me know.

Reply:
Hmm, that seems like a good shout actually considering the snippet is included on WP posts/pages etc as opposed to being a Site Kit only thing. Thanks for explaining!

I should also explain though, my main point about readability was due to the abbreviation of `subscriptions` to `s`. It's losing the meaning of the variable. How about adding a brief comment to clarify it?

Reply:
Not sure about adding a comment, just renamed it back to `subscriptions` 👍.

Reply:
Sounds good to me, nice one.